submarine-workbench/workbench-web-ng/src/app/interfaces/experiment-spec.ts

```diff
@@ -53,5 +53,5 @@ export interface ExperimentSpec {
   meta: ExperimentMeta;
   environment: EnvironmentSpec;
   spec: Specs;
-  code: Code;
+  code?: CodeSpec;
```

Comment:
`code?` Is there a problem here?

Reply:
"?" hear mean optional, so we could set input null if we don't want to sync up code from Github.

Reply:
OK.